Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -225,30 +234,31 @@ open class QuotedChatMessageView: _View, ThemeProvider, SwiftUIRepresentable {
     /// the attachments preview of the message, or if you want to support your custom attachment.
     /// - Parameter message: The message that contains all the attachments.
     open func setAttachmentPreview(for message: ChatMessage) {
+        let currentMessageText = textView.text
         if let filePayload = message.fileAttachments.first?.payload {
             attachmentPreviewView.contentMode = .scaleAspectFit
             attachmentPreviewView.image = appearance.images.fileIcons[filePayload.file.type] ?? appearance.images.fileFallback
-            textView.text = message.text.isEmpty ? filePayload.title : message.text
+            textView.text = message.text.isEmpty ? filePayload.title : currentMessageText
```

Comment:
Hm, shouldn't we check whether `textView.text.isEmpty` instead? This feels a bit strange.

Reply:
Yes that could be a possiblity as well

Reply:
@martinmitrevski I made this more clear now